core/parentchain/block-importer/Cargo.toml

```diff
@@ -0,0 +1,61 @@
+[package]
+name = "itc-parentchain-block-importer"
+version = "0.8.0"
+authors = ["Integritee AG <hello@integritee.network>"]
+edition = "2018"
+resolver = "2"
+
+[dependencies]
+# sgx dependencies
+sgx_types = { branch = "v1.1.4-testing", git = "https://github.com/apache/teaclave-sgx-sdk.git" }
+sgx_tstd = { branch = "v1.1.4-testing", git = "https://github.com/apache/teaclave-sgx-sdk.git", optional = true }
+
+# local dependencies
+ita-stf = { path = "../../../app-libs/stf", default-features = false }
+itc-parentchain-indirect-calls-executor = { path = "../indirect-calls-executor", default-features = false }
+itc-parentchain-light-client = { path = "../light-client", default-features = false }
+itp-extrinsics-factory = { path = "../../../core-primitives/extrinsics-factory", default-features = false }
+itp-ocall-api = { path = "../../../core-primitives/ocall-api", default-features = false }
+itp-settings = { path = "../../../core-primitives/settings" }
+itp-stf-executor = { path = "../../../core-primitives/stf-executor", default-features = false }
+itp-types = { path = "../../../core-primitives/types", default-features = false }
+
+# sgx enabled external libraries
+thiserror_sgx = { package = "thiserror", git = "https://github.com/mesalock-linux/thiserror-sgx", tag = "sgx_1.1.3", optional = true }
+
+# std compatible external libraries (make sure these versions match with the sgx-enabled ones above)
+thiserror = { version = "1.0", optional = true }
+
+# no-std compatible libraries
+codec  = { package = "parity-scale-codec", version = "2.0.0", default-features = false, features = ["derive"] }
+log = { version = "0.4", default-features = false }
+sp-runtime = { version = "4.0.0-dev", default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "master"}
+#beefy-merkle-tree = { version = "4.0.0-dev", default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "master", features = "keccak" }
+#remove as soon as we can import beefy-merkle-tree:
+tiny-keccak = { version = "2.0.2", features = ["keccak"] }
+
+[features]
+default = ["std"]
+std = [
+    "codec/std",
```

Comment:
I agree we should order them alphabetically, but we might want to introduce the same commenting & grouping style as above.

I.e group` # local`, group `# crates.io` and so on. This will make it easier to see if all libraries are included.